bundles/io/org.eclipse.smarthome.io.rest.test/src/test/groovy/org/eclipse/smarthome/io/rest/test/filter/CorsFilterTest.groovy

```diff
@@ -34,6 +34,8 @@ class CorsFilterTest {
     private static final String HTTP_GET_METHOD = "GET"
     private static final String HTTP_OPTIONS_METHOD = "OPTIONS"
 
+    private static final String CONTENT_TYPE_HEADER = "content-type";
```

Comment:
If we want to use javax.ws.rs.core (as it is done by  ui.icon, automation.rest, core.id, rest.core, rest, rest.sse, rest.sitemap) we can use the constant field http://docs.oracle.com/javaee/7/api/javax/ws/rs/core/HttpHeaders.html#CONTENT_TYPE